addons/project/static/tests/project_models.js

```diff
@@ -6,11 +6,18 @@ export class ProjectProject extends models.Model {
 
     name = fields.Char();
     is_favorite = fields.Boolean();
+    date = fields.Date();
```

Comment:
```suggestion
    date = fields.Date(string="Expiration Date");
```
To avoid confusion, we should give the string.